Woof/WoofTests/ViewModel/DetailPetSitterViewModel/DetailPetSitterViewModel+CopyToClipboardTextTests.swift

```diff
@@ -0,0 +1,59 @@
+import XCTest
+
+// swiftlint:disable:next type_name
+final class DetailPetSitterViewModelCopyToClipboardTextTests: XCTestCase {
+    func testThePassedTextIsCopiedToClipboard() {
+        // Given
+        let textToCopy = "a"
+
+        // When
+        testViewModel.copyToClipboardText(textToCopy)
+
+        // Then
+        XCTAssertEqual(clipboard, textToCopy)
```

Comment:
While reading the code, I might think that you compare a clipboard with the text, but that's not true.

You compare text saved in the system-wide clipboard with another text.

Does that make sense?

Reply:
Fixed